Update panw documentation

mjwolf opened the pull request.

mjwolf commented:
This is what the new readme looks like in Kibana. I didn't realize the comments will be rendered, I'll remove all of them before this is done

![mw-dev kb us-central1 gcp foundit no_app_integrations_detail_panw-5 2 2_overview](https://github.com/user-attachments/assets/526d1cc2-09e8-4733-934c-d882f322f3a2)

haetamoudi commented:
Looks good! 

Looking at the `How do I deploy this integration`, I initially did not understand that `To collect logs via syslog over TCP`, `To collect logs via log file` etc.. were parts of the `5.`. What about updating the indentation like:

----
## How do I deploy this integration?

### Collect logs via syslog

To configure syslog monitoring, please follow the steps mentioned in the [_Configure Syslog Monitoring_](https://docs.paloaltonetworks.com/pan-os/10-2/pan-os-admin/monitoring/use-syslog-for-monitoring/configure-syslog-monitoring).

### Collect logs via log file
To configure log file monitoring, please follow the steps mentioned in [_Configure Log Forwarding_](https://docs.paloaltonetworks.com/pan-os/10-2/pan-os-admin/monitoring/configure-log-forwarding).

### Enable the integration in Elastic

1. In Kibana navigate to **Management** > **Integrations**.
2. In the search bar, type **Palo Alto Next-Gen Firewall**.
3. Select the **Palo Alto Next-Gen Firewall** integration and add it.
4. If needed, install Elastic Agent on the systems which receive syslog messages or log files.
5. Enable and configure only the collection methods which you will use.
    - **To collect logs via syslog over TCP**, enter the following details for the syslog receiver that will be monitored:
        - Syslog Host
        - Syslog Port

    - **To collect logs via syslog over UDP**, enter the following details for the syslog receiver that will be monitored:
        - Syslog Host
        - Syslog Port

    - **To collect logs via log file**, please enter the following details for the log file receiver:
        - Paths

6. Press **Save Integration** to begin collecting logs.
----

kgeller commented:
I think this is looking really awesome so far! 

I agree with @haetamoudi about tweaking the indentation for the deploy steps. 

Then I think we just add a small blurb the scaling section, maybe some generic guidelines for scaling tcp/udp while we finalize that copy with docs. And maybe there is something we could use from the [benchbuilder](https://github.com/elastic/ingest-dev/blob/main/performance/use_cases/bb/integrations/panw.yaml) initiative to mention in terms of benchmarking?

benironside commented:
This looks great, thanks @mjwolf! Were any particular sections where it was hard to implement the template? Areas we should be more specific?

mjwolf commented:
@haetamoudi, @kgeller I've updated the bullets for the different input types and added a small section on performance scaling.

I couldn't get the nested bullets to look good, the line spacing is always wrong, so I removed it. I think it's better without them anyway

I think ideally we'd link to external pages for all static content, so then when the content is updated, we don't need to update all the integrations that use the static content.  There isn't really a good page for integration scaling or performance right now. The ingest architecture page is the best one I could find. When we have more appropriate pages, I think we can replace it.

This is a screenshot of the latest version.
![panw-docs](https://github.com/user-attachments/assets/924fb441-e1d9-4cfc-b9b8-04c268a7fed4)
